src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -292,9 +443,12 @@ def _save_export(self, export):
     def create_export(self, fs_name, cluster_id, pseudo_path, read_only, path):
         try:
             if not self.check_fs(fs_name):
-                return -errno.EINVAL,"", "Invalid CephFS name"
+                raise Exception("Invalid CephFS name")
```

Comment:
why do you want the end user to see this traceback, and not a friendly error message?
```
return -errno.ENOENT, "", f"volume '{fs_name}' not found"
```

Reply:
I was catching this exception and it was not giving any traceback. I have changed it to return error message directly.